deploy/solutions/minio/HwameiStor-MinIOSolutionReadMe.txt

```diff
@@ -0,0 +1,185 @@
+
+HwameiStor-MinIO Cloud Native Solution
+
+1. Solution Introduction
+
+HwameiStor is a high availability local storage system for cloud-native stateful workloads. It creates a local storage resource pool for centrally managing all disks such as HDD, SSD, and NVMe. It uses the CSI architecture to provide distributed services with local volumes and provides data persistence capabilities for stateful cloud-native workloads or components.
+
+MinIO is a High Performance Object Storage released under GNU Affero General Public License v3.0. It is API compatible with Amazon S3 cloud storage service. Use MinIO to build high performance infrastructure for machine learning, analytics and application data workloads.
+
+MinIO is used as the main storage for cloud-native applications. Compared with traditional object storage, cloud-native applications require higher throughput and lower latency. These are the performance indicators that MinIO can achieve, with read/write speeds of up to 183 GB/sec and 171 GB/sec.
+
+The ultimate high performance of MinIO is inseparable from the underlying storage platform. Local storage has the highest read and write performance among many storage protocols, which can undoubtedly provide performance guarantee for MinIO. HwameiStor is a storage system that meets the requirements of the cloud-native era. It has the advantages of high performance, high availability, automation, low cost, and rapid deployment, and can replace expensive traditional SAN storage.
```

Comment:
```suggestion
The ultimate high performance of MinIO is inseparable from the underlying storage platform. Local storage is a good choice with the competitive read and write performance, which can undoubtedly provides performance guarantee for MinIO. HwameiStor is such a local storage system that waves with the cloud-native era. High performance, high availability, automation, low cost, rapid deployment, and more benefits attract you to replace those expensive and traditional storage schemes such as SAN (Storage Area Network).
```

Reply:
"Local storage has the highest read and write performance among many storage protocols, which can undoubtedly provide performance guarantee for MinIO."

Does it mean Hwameistor's Local Storage?